src/Consumer.php

```diff
@@ -313,15 +321,16 @@ protected function getWorker(string $queueName): QueueWorkerInterface {
    * @return bool
    *   Did it ?
    */
-  protected function hitIterationsLimit(int $maxIterations, int $iteration) {
+  protected function hitIterationsLimit(int $maxIterations, int $iteration): bool {
```

Comment:
Much as I regret it, result type hints have been voted against in D8 coding standards, although static type parameter hints have been allowed. So...